package.json

```diff
@@ -17,13 +17,12 @@
     "node": ">=8.0.0"
   },
   "dependencies": {
-    "hoek": "5.x.x",
-    "ioredis": "3.x.x"
+    "ioredis": "3 - 4"
```

Comment:
Ok, here’s the reason you’re still using the event to wait for ioredis to connect. Let’s updated to ioredis v4 and leave v3 behind.